examples/todo-intermediate/src/models/todo.model.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/example-todo
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Entity, property, model} from '@loopback/repository';
+
+@model()
+export class Todo extends Entity {
+  @property({
+    type: 'number',
+    id: true,
+  })
+  id?: number;
+
+  @property({
+    type: 'string',
+    required: true,
+  })
+  title: string;
+
+  @property({
+    type: 'string',
+  })
+  desc?: string;
+
+  @property({
+    type: 'boolean',
+  })
+  isComplete: boolean;
+
+  @property({
+    type: 'string',
+  })
+  remindAtAddress: string; // address,city,zipcode
```

Comment:
Let's remove `remindAtAddress` and `remindAtGeo` properties, they are not relevant to HasMany relation.